build/attestations/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Build attestations
+keywords: build, attestations, sbom, provenance
+description: >
+  Introduction to SBOM and provenance attestations with Docker Build; what they
+  are and why they exist
+---
+
+Build attestations describe how an image was built, and what it contains. The
+attestations are created at build-time by BuildKit, and attach to the final
+image as metadata.
+
+The purpose of attestations is to make it possible to inspect an image and see
+where it comes from, who created it and how, and what it contains. This enables
+policy engines to evaluate and validate an image.
```

Comment:
Also worth including a mention about making it easier for human readers to access this information to make educated decisions about the supply chain security of the chain?